clojure/src/rabbitmq/tutorials/rpc_client.clj

```diff
@@ -0,0 +1,42 @@
+(ns rabbitmq.tutorials.rpc-client
+  (:require [langohr.core :as lc]
+            [langohr.channel :as lch]
+            [langohr.queue :as lq]
+            [langohr.basic :as lb]
+            [langohr.consumers :as lcons])
+  (:import [com.rabbitmq.client QueueingConsumer]))
+
+(def ^{:const true} q "rpc_queue")
+
+(defn correlation-id-equals? [correlation-id d]
```

Comment:
Please move the arguments vector to the next line (I know we don't have a Clojure code style guide but I feel strongly about this).